cypress/e2e/LoginPage.cy.ts

```diff
@@ -22,7 +22,7 @@ describe('Login page', () => {
 
     cy.get('[data-cy="sign-in-button"]').click();
 
-    cy.get('[data-cy="account-button"]').should('have.text', 'Account');
+    cy.url().should('not.include', '/login');
```

Comment:
I like the previous assertion better. Users don't care about what URL they are on; they care about what the see.